backend/hardware/gpiochip/gpiod.py

```diff
@@ -1,13 +1,13 @@
-from hardware.gpiochip.base import GPIOChip, LINE_REQ_DIR_OUT, LINE_REQ_DIR_IN
+from hardware.gpiochip.base import GPIOChip, LINE_REQ_DIR_OUT
 import gpiod
 import logging
 
 
 class GPIODChip(GPIOChip):
-    def __init__(self, args):
```

Comment:
I changed all the instances of `args` to a more appropriate config related value in each circumstance. Specifically in my experience the usage of `args` in method definitions is usually as a pass through of all argument and key name arguments when calling a parent class via something like `super()` ex:

``` python
class SomeClass(ParentClass):

    def __init__(*args, **kwargs):
        super().__init__(*args, **kwargs)
        # Do some other child class stuff
```